twisted/internet/interfaces.py

```diff
@@ -2256,6 +2256,28 @@ def acceptableProtocols():
 
 
 
+class ITLSContextFactory(Interface):
```

Comment:
minor comment

since this is returning and OpenSSL context, maybe it should be named ITLSOpenSSLContextFactory...

We can reserve the ITLSContextFactory name for a future backend independent context created by python cryptography.